pages/common/eza.md

```diff
@@ -0,0 +1,36 @@
+# eza
+
+> eza is a modern, maintained replacement for ls, built on exa.
```

Comment:
```suggestion
> Modern, maintained replacement for ls, built on exa.
```

Reply:
Looks good :+1: I just copied from the eza readme